data/devices/logitech-g502-x-wireless.device

```diff
@@ -0,0 +1,4 @@
+[Device]
+Name=Logitech G502 X Wireless
+DeviceMatch=usb:046d:c098;usb:046d:c547
```

Comment:
`c547` is a receiver PID, it is also used in other devices and would cause conflicts, please, remove it. :)
```suggestion
DeviceMatch=usb:046d:c098
```